signbank/dictionary/migrations/0042_new_video_type_choices.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.23 on 2024-01-09 05:58
+import random
+
+from django.db import migrations
+from django.conf import settings
+
+
+def get_or_create_tags(apps, schema_validation):
+    FieldChoiceModel = apps.get_model("dictionary", "FieldChoice")
+    machine_values = FieldChoiceModel.objects.all().values_list("machine_value", flat=True)
+    random_value = random.randint(0, 99999999)
+    while random_value in machine_values:
+        random_value = random.randint(0, 99999999)
+    FieldChoiceModel.objects.get_or_create(field='video_type', english_name='validation', machine_value=random_value)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dictionary', '0041_named_tags'),
+        ('video', '0004_glossvideo_video_type')
+    ]
+
+    operations = [
+        migrations.RunPython(get_or_create_tags, reverse_code=migrations.RunPython.noop)
```

Comment:
Would the reverse be to remove the fieldchoice model instance?

Reply:
yeah I was just thinking about this a bit more. It might be better to remove it on reverse. The only case I see that would cause the reverse code to run if unapplying these code changes.